### DISCUSSION TITLE
Assign Local Keycloak Groups to federated Users from LDAP

### DISCUSSION THREAD

**Discussion by jeanfortheweb** [keycloak/keycloak#25721] [Q&A]
Hello guys,

I was wondering if it is possible to assign local groups from keycloak to federated users from ldap. Since the federated users have a local unique id i guessed that it should be possible.

The idea behind this is to keep group management out of ldap (we don't sync the users to keycloak) because the users inside ldap are all over the place and do not allow us to structure them as we need them for a certain app.

However, whenever i try to assign a local keycloak group to an ldap user, i just get an error message which simply says nothing.

I was wondering if you guys have any idea how to solve this.

In the end, i need the ability to assign certain groups to certain ldap users which will end up in the token for our app.

Thanks for any help!

Johnny

**Comment by RJanssen81** [comment]
Did you already found a solution/answer to your question, I am having the same situation.

**Reply by LostInTheSauce307** [reply]
Did you find any solution to this?

**Comment by LostInTheSauce307** [comment]
I am having the same issue.  Has this been fixed by anyone?
